src/k8s/pkg/utils/experimental/snapdconfig/k8s_to_snapd.go

```diff
@@ -13,7 +13,7 @@ import (
 func SetSnapdFromK8sd(ctx context.Context, config apiv1.UserFacingClusterConfig, snap snap.Snap) error {
 	var sets []string
 	for key, cfg := range map[string]any{
-		"meta":          Meta{Orb: "snapd", APIVersion: "1.30"},
+		"meta":          Meta{Orb: "k8sd", APIVersion: "1.30"},
```

Comment:
AFAIK, this function is called on `x_snapd_config reconcile` when `meta.orb=k8sd`, however after reconciling we set it back to `meta.orb=snapd` which will override the initial option. Not sure if this is a bug or this is intended, I need more context on this

Reply:
revert

Reply:
`meta.orb` is used to denote which source of truth to use. when k8sd updates the configs, we set it to `snapd`, so that any `snap set` afterwards are applied